dapps/archgregator.json

```diff
@@ -9,10 +9,10 @@
   "phone": "",
   "location": "",
   "summary": "",
-  "logo": "",
+  "logo": "dapps/assets/ArchGregator.png",
```

Comment:
this is the wrong path for the logo... when uploaded in the github repository it. should be only the file name... remove "dapps/assets/"

Reply:
Okay, I'll update. I had included just Archgregator.png originally but it didn't seem to pick up the image so was trying this way. Will try again.